yarn.lock

```diff
@@ -14457,4 +14457,4 @@ zip-stream@^1.2.0:
     archiver-utils "^1.3.0"
```

Comment:
Can't we also drop preact and preact-compat from `package.json` and `webpack.js` ?

Reply:
Yeah, wanted to do that in a followup because preact is now used as react for the calendar.